packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.mdx

```diff
@@ -0,0 +1,35 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as viewerStories from './rich-text-viewer.stories';
+import * as mentionUserStories from '../../../rich-text-mention/users/tests/rich-text-mention-users.stories';
+import * as userMappingStories from '../../../mapping/user/tests/mapping-user.stories';
+
+<Meta of={viewerStories} />
+
+<Title of={viewerStories} />
+<Description of={viewerStories} />
+
+<DocsStory of={viewerStories.richTextViewer} expanded={false} />
+<Controls of={viewerStories.richTextViewer} />
+
+# Usage Docs
+
+## User Mention Usage
+
+To enable support for `@mention` in the `nimble-rich-text-viewer` component, the following configuration and mapping components should be added as children to convert the input markdown into a mention node, rendering a user name. Also, see [Client Usage Guidance on Filtered Users](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/rich-text/specs/mention-hld.md#client-usage-guidance-on-filtered-users-1) for instructions on loading the mapping elements into the viewer component. Follow the document for specific APIs in the below configuration and mapping elements.
```

Comment:
I think we should provide the guidance about which users to include inline within this doc rather than linking to the spec. It can also be more succinct: simply state that every user mentioned in the markdown should have a corresponding user mapping.

Reply:
Updated as per the suggestion. Also, I feel keeping the link to the spec doc as is in the storybook so that the client can find more details about how can we get the mentioned users list from the markdown string. Please let me know if we should not link the doc to the storybook.

Reply:
Linking to the spec for additional information is ok.